src/Aspire.Hosting.MongoDB/MongoDBContainerImageTags.cs

```diff
@@ -7,8 +7,8 @@ internal static class MongoDBContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "library/mongo";
-    public const string Tag = "7.0.8";
+    public const string Tag = "7.0";
     public const string MongoExpressRegistry = "docker.io";
-    public const string MongoExpressImage = "mongo-express";
-    public const string MongoExpressTag = "1.0.2-20";
+    public const string MongoExpressImage = "library/mongo-express";
+    public const string MongoExpressTag = "1.0";
```

Comment:
mongo-express seems to have weird tagging as 1.0 now is NOT 1.0.2-20 FYI here. Not sure we change this based on the principle, but an FYI on how it isn't always mapping

Reply:
@eerhardt there are `1.0-20` and `1.0-18` tags that let you choose which version of Node. You can use `1.0` if you don't mind about the underlying Node version. The default now is Node 18 but I suspect that will change in the future.

Reply:
Thanks for the info. I think sticking to `1.0` makes the most sense for us. As the `1.0` tag gets updated for newer Node versions, we will automatically pick up the new version.

Reply:
Actually I think there might be a bug with 1.0 around auth. Make sure you test it.

Reply:
What bug are you thinking? I used the mongo playground app, and was able to hit the mongo-express UI and saw the containers inside of it.